orc-rt/include/orc-rt-c/orc-rt.h

```diff
@@ -0,0 +1,27 @@
+/*===- orc-rt-c/orc-rt.h - Placeholder header for orc-rt ----------*- C -*-===*\
+|*                                                                            *|
+|* Part of the LLVM Project, under the Apache License v2.0 with LLVM          *|
+|* Exceptions.                                                                *|
+|* See https://llvm.org/LICENSE.txt for license information.                  *|
+|* SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception                    *|
+|*                                                                            *|
+|*===----------------------------------------------------------------------===*|
+|*                                                                            *|
+|* Placeholder header for initial orc-rt checkin                              *|
+|*                                                                            *|
+\*===----------------------------------------------------------------------===*/
```

Comment:
Is there any particular reason for using `/* */` comments rather than `//`?  We don't support pre-C99 in LLVM and we use `//` in other C sources elsewhere, see for example https://github.com/llvm/llvm-project/blob/c876d719ef5b10128eca6d8677068248b4831017/compiler-rt/lib/builtins/absvdi2.c.

Reply:
No reason -- I just cribbed the old style license header from one of the llvm-c headers. I've updated to C99 style comments.